project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.hanhuy.sbt" % "android-sdk-plugin" % "1.5.19")
+addSbtPlugin("org.scala-android" % "sbt-android" % "1.6.17")
```

Comment:
You know, just wondering if `sbt-android` made some changes in `1.6.x` that could break the other modules